src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/Resources/SourceGenerator_RazorFiles_Works/Pages_Index_razor.g.cs

```diff
@@ -0,0 +1,22 @@
+﻿#pragma checksum "/0/Pages/Index.razor" "{ff1816ec-aa5e-4d10-87f7-6f4963833460}" "6b5db227a6aa2228c777b0771108b184b1fc5df3"
```

Comment:
📝 This line is the only change to this file relative to the original string. It's not clear to me whether a full build with the source generator running will use an absolute path here or a relative path here. If it's a relative path, I'm not sure where it's derived from.